src/nvim/os/fs.c

```diff
@@ -272,43 +293,55 @@ static bool is_executable(const char_u *name)
 static bool is_executable_in_path(const char_u *name, char_u **abspath)
   FUNC_ATTR_NONNULL_ARG(1)
 {
-  const char *path = os_getenv("PATH");
-  if (path == NULL) {
+  const char *path_env = os_getenv("PATH");
+  if (path_env == NULL) {
+    if (p_verbose >= 42) {
+      smsg("os_can_exe false 3");
+    }
     return false;
   }
 
-  size_t buf_len = STRLEN(name) + STRLEN(path) + 2;
+#ifdef WIN32
+  // Prepend ".;" to $PATH.
+  size_t pathlen = strlen(path_env);
+  char  *path    = memcpy(xmallocz(pathlen + 3), ".;", 2);
+  memcpy(path + 2, path_env, pathlen);
+#else
+  char *path = xstrdup(path_env);
+#endif
```

Comment:
For Windows, prepend `".;"` to PATH, [as Vim does](https://github.com/vim/vim/blob/c4a249a736d40ec54794827ef95804c225d0e38f/src/os_win32.c#L1916                                              
).